templates/partials/global/search-nav.html

```diff
@@ -1,19 +1,27 @@
-<a class="searchtab{{#if isFilterAll}} searchtab--active{{/if}}" href="{{links.filterAll}}">
+<a class="searchtab{{#if isFilterAll}} searchtab--active{{/if}}"
+  {{#unless filterAllDisabled}}href="{{links.filterAll}}"{{/unless}}
+  {{#if filterAllDisabled}}aria-disabled="true"{{/if}}>
```

Comment:
the two conditions above can be done with a if else:
```
{{#if filterAllDisabled}}
  aria-disabled="true"
{{else}}
  href="{{links.filterAll}}"
{{#if}}
>
```
@tobystokes do you have any preference?